3.3/.sti/bin/assemble

```diff
@@ -1,23 +1,66 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
```

Comment:
I must have missed that, what you guys decided about assemble and run functions. I thought they will be placed in-line at the end of this file, to get rid of unnecessary functions.